extensions/kind/src/util.ts

```diff
@@ -97,7 +97,7 @@ export function runCliCommand(
     let stdOut = '';
     let stdErr = '';
     let err = '';
-    let env = Object.assign({}, process.env); // clone original env object
+    const env = options?.env ?? process.env;
```

Comment:
This is slightly different from the original behavior. 
It first init env with process.env and then merge it with options.env. 
It seems to me that you only pick options.env if it's not null, otherwise process.env